README.md

```diff
@@ -137,6 +137,17 @@ message = client.messages.create(to="+12316851234", from_="+15555555555",
                                  body="Hello there!")
 ```
 
+### Enable Debug Logging
+
+View the API request and response data in your console:
+
+```python
+import logging
+
+client = Client(account, token)
+logging.basicConfig(level=logging.INFO)
```

Comment:
They may not want this level of logging everywhere. Think it's better to be specific to our default rest client.